public/electron.js

```diff
@@ -176,6 +177,11 @@ ipc.on('syncPatientId', (event) => {
   event.returnValue = process.env.REACT_APP_PATIENT_ID
 })
 
+// Get Study Id from environment
```

Comment:
should the syncing of PatientID and study ID be two different events? or should/could we do this in one go.  I also wonder since these aren't actually build time react env vars, if we should switch up the naming? maybe `BEADS_TASK_PATIENT_ID` or more generically to be used across many tasks `TASK_APP_PATIENT_ID` (and `STUDY_ID` respectively) @mirestrepo what do you think?

Reply:
I'ma bit confused here as to what is the expectation for Electron? Are we now requiring always `PARTICIPANT_ID` and `STUDY_ID`?  In firebase we are requiring both as all data lives in the same database. I don't know the organization in the file-system, but in the "clinic" case, I feel like the scenario of having the STUDY_ID come from and ENV and the PARTICIPANT_ID be input at run time is a possibility?

`syncCredentials()` sounds reasonable to me, and naming the variables differently as well. What about just `PARTICIPANT_ID` and `STUDY_ID`? Any good reason for a prefix?